### DISCUSSION TITLE
Where to find and edit English phrases 'On this page', 'Connected pages', 'Search', etc.

### DISCUSSION THREAD

**Discussion by zanodor** [oleeskild/obsidian-digital-garden#509] [General]
...if someone wants to translate these phrases to one's own language?  

I know this is not supported and one may have to rewrite these parts on updates but I'd still like to know.  
I looked around in .js and .json files and couldn't find these phrases.

Thanks

**Comment by Neutronicsenthusiast064** [comment]
Hello,

Check here:
![image](https://github.com/oleeskild/obsidian-digital-garden/assets/30866612/b81b395c-ca85-4c4c-9ac7-faed8ec4cc2b)
You will find different .njk files and you will find what you search.

For example, if you want to modify the table of content phrase "On this page", go to `sidebar.njk`, line 17.
![image](https://github.com/oleeskild/obsidian-digital-garden/assets/30866612/4e81bbe5-fbee-45bb-89a9-676b240f5153)

**Reply by zanodor** [reply]
Will do, thanks a bunch.
